packages/dropdowns/src/elements/Menu/Menu.tsx

```diff
@@ -22,18 +22,23 @@ import {
 import { MenuContext } from '../../utils/useMenuContext';
 
 interface IMenuProps extends HTMLAttributes<HTMLUListElement> {
-  /** See Popper [documentation](https://popper.js.org/docs/v2/modifiers/) for details */
+  /** Sets the [Popper](https://popper.js.org/docs/v2/modifiers/) modifiers */
```

Comment:
I think the link should either surround "modifiers" or both "Popper modifiers".

Reply:
Fixed in latest push.